### PR TITLE
feat: Initial commit

### PR DESCRIPTION
**Pull request by michaeltaranto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+
+#### filepath (string, default: '<cwd>/.gitignore')
+Path to the `.gitignore` file to be used.
+
+#### ignores (?Array\<string>)
+An array of strings to ensure are present in the specified `.gitignore` file. Any ignore that already exists in the file will be appended with the [suffix](#suffix). All new ignores will be appended to the bottom of the file.
+
+<a id="suffix">
+
+#### suffix (?string, default: "managed")
```
Comment (markdalgleish) [inline README.md]: 🚲🏡: How about `comment`? Not sure I'd call this a suffix.

Also, I feel like a sensible default would actually be no comments at all, but I can see that's debatable.

**Review by mattcompiles — APPROVED** [review]
Noice 🚀 📦

File: package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "gitignore-ensure",
+  "version": "0.0.0-development",
+  "description": "Ensure the presence of paths within a project's gitignore",
```
Comment (markdalgleish) [inline package.json]:
```suggestion
  "description": "Ensure the presence of patterns within a project's gitignore",
```

File: test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+`);
+  });
+
+  it('error if file not found', () => {
+    const manage = () =>
+      managedIgnore({
+        filepath: 'output/notfound'
+      });
+
+    expect(manage).toThrowErrorMatchingSnapshot();
```
Comment (markdalgleish) [inline test/index.test.js]: Should this be `toThrowErrorMatchingInlineSnapshot`?

**Review by mattcompiles — APPROVED** [review]
🚀

**Review by markdalgleish — APPROVED** [review]

**Pull request merged**